nexus-sdk-js/README.md

```diff
@@ -0,0 +1,8 @@
+# SDK for nexus
+
+## Develop
+
+- Build: `yarn build`
```

Comment:
Any reason to use yarn here while our standard tool is npm?

Reply:
Emm, I think that's because I did that before we came up with our standards. I'll update for consistency

Reply:
fixed in c3ab83172e987c340bf38b3f1931fe43322f1a50